extress.js

```diff
@@ -14,27 +14,13 @@ const Extress = {
     buildTree(app._router.stack);
     axios.post('http://localhost:4050/tree', Extress.tree); //put this onto client side
   },
-  durationArr: [],
+  durationObj: {},
   startTime: null,
-  perfData: {
-    routeDuration: null,
-    method: null,
-    route: null,
-    min: null,
-    max: null,
-    avg: null
-  },
+  perfData: [],
 
   reset: () => {
-    Extress.durationArr = [];
-    Extress.perfData = {
-      routeDuration: null,
-      method: null,
-      route: null,
-      min: null,
-      max: null,
-      avg: null
-    };
+    Extress.durationObj = {};
+    Extress.perfData = [];
```

Comment:
`Extress.perfData` is no longer used, please remove.

Reply:
Removed.